pages/common/aws-pricing.md

```diff
@@ -0,0 +1,24 @@
+# aws pricing
+
+> Centralized and convenient way to programmatically query Amazon Web Services for services, products, and pricing information.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/pricing/>.
+
+- Get a list of service codes to query later for:
+
+`aws pricing describe-services --region {{us-east-1}}`
+
+- Get a a list of attributes for a given service code:
```

Comment:
`get` is more for programming that terminal
```suggestion
- List attributes for a given service code in a specific region:
```

Reply:
Totally agree!